Sources/Private/Experimental/Layers/ShapeLayer.swift

```diff
@@ -19,7 +19,11 @@ final class ShapeLayer: BaseCompositionLayer {
     let groupItems = shapeLayer.items.compactMap { $0 as? Group }
     let otherItems = shapeLayer.items.filter { !($0 is Group) }
 
-    for group in groupItems {
+    // Groups are listed from front to back,
+    // but `CALayer.sublayers` are listed from back to front.
+    let groupsInZAxisOrder = groupItems.reversed()
```

Comment:
He's alive!

<img width="811" alt="image" src="https://user-images.githubusercontent.com/1811727/148591922-796bbff1-c689-4672-a6ea-341b5c9192bd.png">

Reply:
This change also fixes issue with `IconTransitions`: 

<img width="601" alt="image" src="https://user-images.githubusercontent.com/1811727/148591987-f1a614f9-c27b-4a66-a8a6-363b9e6404a5.png">